articles/cognitive-services/Acoustics/what-is-acoustics.md

```diff
@@ -55,11 +55,10 @@ The Project Acoustics editor plugin is available for the following platforms:
 ## Download
 * [Project Acoustics Unity plugin and samples](https://www.microsoft.com/en-us/download/details.aspx?id=57346)
 * [Project Acoustics Unreal & Wwise plugins and samples](https://www.microsoft.com/download/details.aspx?id=58090)
-  * For Xbox binaries and support, contact us via the Sign Up form below
+  * For Xbox binaries and other support, contact us via the forum: (https://github.com/microsoft/ProjectAcoustics/issues)

```

Comment:
I think that you should use the markdown link format:

i.e.
"[forum](https://github.com/microsoft/ProjectAcoustics/issues)."